probe-rs/src/debug/mod.rs

```diff
@@ -236,8 +245,22 @@ impl<'debuginfo, 'probe, 'core> Iterator for StackFrameIterator<'debuginfo, 'pro
 
             use gimli::read::RegisterRule::*;
 
-            self.registers[i] = match unwind_info.register(gimli::Register(i as u16)) {
-                Undefined => None,
+            let register_rule = unwind_info.register(gimli::Register(i as u16));
+
+            log::trace!("Register {}: {:?}", i, &register_rule);
+
+            self.registers[i] = match register_rule {
+                Undefined => {
+                    // If we get undefined for the LR register (register 14) or any callee saved register,
```

Comment:
Maybe add a log statement here too, or do you think this is not really an issue?

Reply:
Why do you want a log statement here? I think the log statement two lines earlier is enough.

Reply:
Ohwow I am stupid, I didn't get that you actually log the enum :)